frontend/public/locales/en/eligibility.json

```diff
@@ -0,0 +1,68 @@
+{
+  "breadcrumbs": {
+    "canada-ca": "Canada.ca",
+    "benefits": "Benefits",
+    "dental-coverage": "Dental coverage",
+    "canadian-dental-care-plan": "Canadian Dental Care Plan"
+  },
+  "type-of-application": {
+    "page-title": "Type of application",
+    "form-instructions": "Who is this application for?",
+    "radio-options": {
+      "personal": "I am applying for myself",
+      "delegate": "I am applying on behalf of someone else"
+    },
+    "error-message": {
+      "Required": "Applicant type must be provided"
+    }
+  },
+  "tax-filing": {
+    "page-title": "Tax filing",
+    "form-instructions": "Have you and/or your spouse/common-law partner (if applicable) filed your tax return for 2023 and have received your Notice of Assessment?",
+    "radio-options": {
+      "yes": "Yes",
+      "no": "No"
+    },
+    "error-message": {
+      "Required": "Tax filing must be provided"
+    }
+  },
+  "date-of-birth": {
+    "page-title": "Date of birth",
+    "description": "Applications for the Canadian Dental Care Plan (CDCP) will open in phases. Your date of birth determines when you can apply.",
+    "form-instructions": "Please enter your date of birth",
+    "month": "Month",
+    "day": "Day",
+    "year": "Year",
+    "error-message": {
+      "day": "Day must be provided",
+      "month": "Month must be provided",
+      "year": "Year must be provided"
+    }
+  },
+  "application-delegate": {
+    "page-title": "Applying on behalf of someone cannot be completed online at this time",
+    "description1": "You must <contactservicecanada>speak to a Service Canada representative</contactservicecanada> to apply on behalf of someone. Please contact Service Canada at 1-833-537-4342 or visit a Service Canada office near you.",
```

Comment:
I would try to avoid using numbered keys here, in case the page content gets reordered at some point.

What about using `contact-representative` for this one?

Reply:
The phone number should be wrapped in a non-breaking span here, too. we don't want the page to render like:

```
Please contact Service Canada at 1-833-
537-4342 or visit a Service Canada office
near you.
```

Reply:
Changed to suggested naming scheme. Added in a nowrap `<span>` for the phone number string.